internal/command/fmt.go

```diff
@@ -407,7 +413,7 @@ func (c *FmtCommand) formatValueExpr(tokens hclwrite.Tokens) hclwrite.Tokens {
 			hasTrailingParen = true
 		}
 	}
-	if isMultiLine && !(hasLeadingParen && hasTrailingParen) {
+	if isMultiLine && (!hasLeadingParen || !hasTrailingParen) {
```

Comment:
Interesting that this conditional is rewritten in a different manner than [this](https://github.com/opentofu/opentofu/pull/2798/files#diff-b7f5b7c669ea7bb97e98d899d204d9e6bf48a00d18cb90109447eca05997704dR235) was in the current PR.
Any particular reason of this difference? I am curious about the reason around having 2 different ways of writing these conditionals.